scripts/release/debian/test_deb_package.py

```diff
@@ -24,5 +24,5 @@
     elif exit_code != 0:
         sys.exit(exit_code)
 
-exit_code = subprocess.call(['{} --junit-xml /azure_cli_test_result/azure-cli-core.xml --pyargs azure.cli.core'.format(pytest_parallel_cmd)], shell=True)
```

Comment:
The concurrent writes mainly occur at https://github.com/Azure/azure-cli/blob/060b414d8ac5e4d8f4fc17cc2e23c2154c762226/src/azure-cli-core/azure/cli/core/_profile.py#L496

Since this sprint will end soon, in the next sprint I will see how I can mock `set_cloud_subscription`.